librz/analysis/var.c

```diff
@@ -11,20 +11,154 @@
 
 #define ACCESS_CMP(x, y) ((st64)((ut64)(x) - (ut64)((RzAnalysisVarAccess *)y)->offset))
 
-static bool storage_equals(RzAnalysisVarStorage *a, RzAnalysisVarStorage *b) {
+RZ_API int rz_analysis_var_storage_cmp(const RzAnalysisVarStorage *a, const RzAnalysisVarStorage *b) {
```

Comment:
@thestr4ng3r as you are who recently changed the handling stack variables, please take a look at the changes of `RzAnalysisVar` storage

Reply:
@imbillow please add Doxygen for this function